CONTRIBUTING.md

```diff
@@ -30,10 +30,11 @@ All contributors are expected to follow the [Rust Code of Conduct].
 ## The Clippy book
 
 If you're new to Clippy and don't know where to start the [Clippy book] includes
-a developer guide and is a good place to start your journey.
+a [developer guide] and is a good place to start your journey.
 
 <!-- FIXME: Link to the deployed book, once it is deployed through CI -->
 [Clippy book]: book/src
+[developer guide]: book/src/development
```

Comment:
Could you maybe update this, to link to the book version instead of the markdown files? :)

Reply:
I'd be happy to, but I can't seem to find the book version. What's the URL, please?

Reply:
The book is quite new and a bit harder to find then I would like. :sweat_smile: Could you also update the `[Clippy book]` link? Here are the links :upside_down_face: 

```md
[Clippy book]: https://doc.rust-lang.org/nightly/clippy/index.html
[developer guide]: https://doc.rust-lang.org/nightly/clippy/development/index.html
```

Reply:
Tanks, done.

I did it by fixing up my commit and force-pushing it. I hope that's OK. It probably does not matter for such a small change, but what's the policy for incorporating PR feedback? Fix up and force push or add additional commits?

Reply:
That's perfect! There is no real policy, enforcing it would be more work than it's worth since we have so many contributors. GitHub used to not displaying force pushes correctly, but now it's a viable alternative. Generally, I do the following:

* On small PRs like this one, just fixup and force push, that's totally fine and allows us to merge the PR directly
* For larger PRs with more changes I create a normal new commit describing the change
* For larger PRs with small changes or suggestions, I create a commit like "Address review feedback" and then squash sequential changes with that one.

It's also fine to add a new commit for every review. If there are multiple/too many commits, we just ask you to squash them right before the merge.